src/editors/EditorContainer.tsx

```diff
@@ -19,220 +19,155 @@ export interface Props<R, K extends keyof R> extends SharedInteractionMasksProps
   firstEditorKeyPress: string | null;
 }
 
-interface State {
-  isInvalid: boolean;
-}
-
-export default class EditorContainer<R, K extends keyof R> extends React.Component<Props<R, K>, State> {
-  static displayName = 'EditorContainer';
-
-  changeCommitted = false;
-  changeCanceled = false;
+export default function EditorContainer<R, K extends keyof R>({
+  rowIdx,
+  column,
+  rowData,
+  width,
+  height,
+  left,
+  top,
+  onCommit,
+  onCommitCancel,
+  scrollLeft,
+  scrollTop,
+  firstEditorKeyPress: key
+}: EditorContainerProps<R, K>) {
+  const editorRef = useRef<Editor>(null);
+  const [isInvalid, setIsInvalid] = useState(false);
```

Comment:
```suggestion
  const [isValid, setValid] = useState(true);
```
We don't need to do this now as it's kind of a deprecated component, but I prefer to avoid negative variable names, they can make it a little bit harder to read the code.